packages/flutter_web_plugins/lib/src/navigation/url_strategy.dart

```diff
@@ -288,16 +298,16 @@ class BrowserPlatformLocation extends PlatformLocation {
   }
 
   @override
-  String get pathname => _location.pathname;
+  String get pathname => _location.pathname ?? _defaultPathname;
```

Comment:
FWIW, in the engine, we are adding a `!` to fix this:

https://github.com/flutter/engine/blob/acaef6f00d78da2f9a8f0fafb36d1c45e6b2513d/lib/web_ui/lib/src/engine/navigation/url_strategy.dart#L268

But I'll leave it up to you what you prefer to do here :)

Reply:
I prefer having a default value, since the dart:html API seems to suggest that there's the chance that `null` may be returned? (maybe they know about some browser quirk that we don't?)